server/index.js

```diff
@@ -5,15 +5,16 @@ const path = require('path');
 var app = express();
 var bodyParser = require('body-parser');
 
-app.use(express.static(__dirname + '/public'));
+app.use(express.static(path.join(__dirname, '../public')));
+console.log(__dirname + '/public');
```

Comment:
Nice - this looks like what I had to do. `path.join()` is always a win in my book.

Reply:
Yeah, I see everyone is doing both app.use and sendfile, which is not needed